skins/pok.scss

```
@@ -0,0 +1,10 @@
+.example {
```

The css selector should similar to the skin field value.
```suggestion
.pok {
```